$NetBSD: patch-CMakeLists.txt,v 1.5 2023/11/06 09:31:15 adam Exp $

Do not install includes in a sub-directory.
Install cmake files into lib/cmake.

--- CMakeLists.txt.orig	2023-10-28 15:23:34.000000000 +0000
+++ CMakeLists.txt
@@ -32,17 +32,17 @@ install(TARGETS ${PROJECT_NAME}
 configure_package_config_file(
     "${PROJECT_SOURCE_DIR}/${PROJECT_NAME}Config.cmake.in"
     "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
-    INSTALL_DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake
+    INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
 )
 
 install(EXPORT ${PROJECT_NAME}Targets
         FILE ${PROJECT_NAME}Targets.cmake
         NAMESPACE ${PROJECT_NAME}::
-        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake)
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 install(FILES "${PROJECT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
               "${PROJECT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake"
-        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake)
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 
 install(FILES ${PROJECT_SOURCE_DIR}/source/utf8.h DESTINATION include)
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/source/utf8 DESTINATION include)
